api/v1alpha1/databasecluster_types.go

```diff
@@ -16,7 +16,7 @@
 package v1alpha1
 
 import (
-	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
while working with Kubernetes client-go having this import using `corev1` makes it more clear which API and package is responsible for the structures you use.